src/components/Dashboard/TodoList.vue

```diff
@@ -874,7 +874,9 @@ export default {
       ).then(async (confirm) => {
         // if we get here, "Delete" was clicked
         if (confirm) {
-          await this.doDeleteDraft(task)
+          // delete without refreshing the dashboard as it triggers an error loading an IA
```

Comment:
Is there some sort of spinner during the delete action? Or does the dialog just stay up?

Reply:
There doesn't seem to be any spinner and the dialog closes after confirmation of delete.

Reply:
I have tested this locally, and while the error dialog no longer displays (because the dashboard is not reloaded), for a second or so I see the old dashboard and it still looks enabled. I think you need to do more here so the user does not get confused by seeing an "active-looking" dashboard.

Reply:
What do you think of leaving that dialog up until the redirection happens? So don't "return" from confirmDeleteIncorporation() ? Or perhaps make the dashboard disabled, or look like it's in loading state?

Reply:
I am also thinking of the options you suggested. I will see what is easiest and simplest to do. thanks!

Reply:
- updated confirm dialog component to allow for the dialog to stay open and added a close() function in the event someone wants to use this and manually close the dialog when some condition is met.

- added a loading indicator when someone hits the yes/confirm button and disabled all the other buttons.

Reply:
Nice.